test/unit/storage/cookiestorage.test.js

```diff
@@ -0,0 +1,183 @@
+import expect from 'expect';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { NotFoundError } from 'kinvey-js-sdk/dist/export';
+import { CookieStorageAdapter } from '../../../src/middleware/src/storage/webstorage';
+
+chai.use(chaiAsPromised);
+chai.should();
+
+function randomString() {
```

Comment:
This function seems to be present in multiple locations, should move it to a utils file.

Reply:
I moved the function to `utils.js` and use it to shared `randomString()` across the unit tests.